units/player.gd

```diff
@@ -9,12 +9,27 @@ const HORIZONTAL_BUFFER = 75
 
 @export var Projectile : PackedScene
 @onready var projectile_spawner = $ProjectileSpawner
+var projectile_effects: Array[ProjectileEffect] = []
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
 	Projectile = load("res://units/projectile.tscn")
+	# Debugging -- these should be added to the list via some user interaction instead.
+	var double_laser = load("res://projectiles/double_laser.tscn")
+	# TODO:: Not even sure if adding to the scene tree here is necessary.
+	#    It's necessary now just to trigger projectile_effect._ready, but the
+	#    utility functions will get called regardless of its presense in the scene
+	#    tree, as long as thy're in the top-level projectile_effects Array.
+	add_child(double_laser.instantiate())
+	projectile_effects.append(double_laser)
```

Comment:
Note:

This is not actually appending anything to the top-level projectile_effects array. Later instances where this is accessed see an empty array for some reason :shrug: